networkx/drawing/tests/test_latex.py

```diff
@@ -220,14 +211,6 @@ def test_basic_tikz():
     )
 
     assert output_tex == expected_tex
-    # print(output_tex)
-    # # Pretty way to assert that A.to_document() == expected_tex
-    # content_same = True
-    # for aa, bb in zip(expected_tex.split("\n"), output_tex.split("\n")):
-    #     if aa != bb:
-    #         content_same = False
-    #         print(f"-{aa}|\n+{bb}|")
-    # assert content_same
```

Comment:
In that case, how about uncommenting it instead and adding the line-by-line checking to the tests? It would be redundant, but it wouldn't have a material effect on the test run-time and preserves the intent of the original comment (it just makes it executable)!

I will push this up to see what you think, but in the end if we prefer the comment over this alternative it's totally fine by me!

Reply:
Okay - I've implemented this proposal in 109af53. To see how it works in practice, you can change a character from the `expected_tex` then run `pytest networkx/drawing/tests/test_latex.py` - the test failure summary should show the character difference in the specific failing line.